src/Microsoft.Azure.WebJobs.Host/Constants.cs

```diff
@@ -9,5 +9,6 @@ internal static class Constants
         public const string DevelopmentEnvironmentValue = "Development";
         public const string ExtensionInitializationMessage = "If you're using binding extensions (e.g. ServiceBus, Timers, etc.) make sure you've called the registration method for the extension(s) in your startup code (e.g. config.UseServiceBus(), config.UseTimers(), etc.).";
         public const string UnableToBindParameterFormat = "Cannot bind parameter '{0}' to type {1}. Make sure the parameter Type is supported by the binding. {2}";
+        public const string UnableToResolveBindingParameterFormat = "Unable to resolve binding parameter '{0}'. Binding expressions must map to either a value provided by the trigger or a property of the value the trigger is bound to, or be a system binding expression(e.g.sys.randguid, sys.utcnow, etc.).";
```

Comment:
Looks like you're missing a couple spaces near/in your parenthesis portion of the string. Change to this (a bit of rewording as well):

"Unable to resolve binding parameter '{0}'. Binding expressions must map to either a value provided by the trigger or a property of the value the trigger is bound to, or must be a system binding expression (e.g. sys.randguid, sys.utcnow, etc.)."